docs/antrea-network-policy.md

```diff
@@ -409,6 +410,81 @@ spec:
         enableLogging: true
 ```
 
+#### ACNP for multicast
+
+ACNP for IGMP query
```

Comment:
To keep same format as other examples, this should be removed, you can merge the two IGMP policies into one:
```
#### ACNP for IGMP traffic

apiVersion: crd.antrea.io/v1alpha1
kind: ClusterNetworkPolicy
metadata:
  name: acnp-drop-igmp-query-report
spec:
...
    ingress:
      - name: dropIGMPQuery
        action: Drop
        from:
        protocols:
        - igmp:
            igmpType: 0x11
            groupAddress: 224.0.0.1
    egress:
      - name: dropIGMPReport
        action: Drop
        from:
        protocols:
        - igmp:
            igmpType: 0x16
            groupAddress: 225.1.2.3

#### ACNP for multicast egress traffic

...
```

Reply:
done